Add comment for indicator filtering behaviour (#130)
msupply-foundation/msupply_docs: content/docs/programs/program_indicators.md

![Indicators edit](/docs/programs/images/indicators_edit.png)

From here you can enter in the data, as configured for each indicator of this program.

<div class='note'>

>     Only indicators which are set to active will show up for any requisition. This includes requisitions made with indicators which were previously active; they will no longer be displayed.

Reviewer: ```suggestion
    Only indicators which are set to active will show up for any requisition. If a historical requisition used an indicator that is no longer active, it not be displayed.
```

That feels a bit simpler to me?

Reviewer 2: it will not be displayed

>     Indicator lines which don't have a value also will not show up in the ui.

Reviewer: ```suggestion
    Indicator lines which don't have a value also will not be shown.
```
